test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
Yes. There is no point in adding that anymore since there are visibility modifiers.